SemanticFormsSelect.hooks.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace SFS;
+
+/**
+ * @license GNU GPL v2+
+ * @since 3.0
+ * @author: Alexander Gesinn
+ */
+class Hooks {
+	/**
+	 * Register Page Forms Input Type
+	 *
+	 * This is attached to the MediaWiki 'ParserFirstCallInit' hook.
+	 *
+	 * @param $parser Parser
+	 * @return bool
+	 */
+	public static function onSemanticFormsSelectSetup ( & $parser ) {
+
+		if ( !defined( 'PF_VERSION' ) ) {
+			die( '<b>Error:</b><a href="https://github.com/SemanticMediaWiki/SemanticFormsSelect/">Semantic Forms Select</a> requires the <a href="https://www.mediawiki.org/wiki/Extension:PageForms">Page Forms</a> extension. Please install and activate this extension first.' );
+		}
+
+		if ( isset( $GLOBALS['wgPageFormsFormPrinter'] )) {
+			$GLOBALS['wgPageFormsFormPrinter']->registerInputType( '\SFS\SemanticFormsSelectInput' );

```

Comment:
`SemanticFormsSelectInput::class` is better, since this is tool friendly

Reply:
I don't get what you suggest here...

Reply:
->registerInputType( \SFS\SemanticFormsSelectInput::class );